core/src/value/primitive.rs

```diff
@@ -604,6 +604,31 @@ impl PrimitiveValue {
         }
     }
 
+    /// Getter method to provide a clean version of a string or multiple strings,
+    /// removing unwanted whitespaces
```

Comment:
Please consider following the documentation structure found in other methods (e.g. `to_str`), by adding a small description of the conversions made, and an example.